<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Editeur</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>478</width>
               <height>252</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTableView" name="tableau"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="hauteurLabel">
              <property name="text">
               <string>Hauteur</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="hauteurSpinbox"/>
            </item>
            <item>
             <widget class="QLabel" name="longueurLabel">
              <property name="text">
               <string>Longueur</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="longueurSpinbox"/>
            </item>
            <item>
             <widget class="QCheckBox" name="variationBox">
              <property name="text">
               <string>Variations</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="updateTabular">
              <property name="text">
               <string>Adapter le tableau</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="getCode">
              <property name="text">
               <string>Générer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Code</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTextEdit" name="codeEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="effacerButton">
            <property name="text">
             <string>Effacer</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu_dition">
    <property name="title">
     <string>Édition</string>
    </property>
    <addaction name="actionAdapter_le_tableau"/>
    <addaction name="actionG_n_rer_le_tableau"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu_dition"/>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdapter_le_tableau">
   <property name="text">
    <string>Adapter le tableau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionG_n_rer_le_tableau">
   <property name="text">
    <string>Générer le code</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
